scheds/rust/scx_layered/src/stats.rs

```diff
@@ -241,13 +247,15 @@ impl LayerStats {
 
         writeln!(
             w,
-            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig={} affn_viol={}",
+            "  {:<width$}  open_idle={} mig={} xnuma_mig={} xllc_mig={} affn_viol={} xlayer_wake={} xlayer_rewake={}",
```

Comment:
Let's break the line so that they end up more or less with similar column width.